app/containers/MapContainer/utils.js

```diff
@@ -57,6 +57,104 @@ export const valueOfCircle = (radius, range, config) => {
   return scale(radius);
 };
 
+export const filterFeatureMaxZoom = (feature, zoom) => {
+  if (
+    feature.properties && feature.properties.marker_max_zoom
+  ) {
+    return zoom <= parseInt(feature.properties.marker_max_zoom, 10);
+  }
+  return false;
+};
+
+// append point countries onto map
+export const getPointLayer = ({ data, config, markerEvents }) => {
+  const layer = L.featureGroup(null);
+  const {
+    indicator, mapOptions, mapSubject, maxValueCountries, styleType, valueToStyle, zoom,
+  } = config;
+  const events = {
+    mouseover: (e) => markerEvents.mouseover ? markerEvents.mouseover(e, config) : null,
+    mouseout: (e) => markerEvents.mouseout ? markerEvents.mouseout(e, config) : null,
+    click: (e) => (markerEvents.click ? markerEvents.click(e, config) : null),
+  };
+
+  const jsonLayer = L.geoJSON(data, {
+    pointToLayer: (feature, latlng) => {
+      const filterFeature = filterFeatureMaxZoom(feature, zoom);
+      const styles = getAreaLayer(feature, indicator, mapOptions, mapSubject, maxValueCountries, styleType, valueToStyle);
```

Comment:
@vbojilova ultimately I am not sure the marker styles will share anything with the area styles except for the fill color so not sure if a common style function is needed here